### PR TITLE
Removed broken links

### PR DESCRIPTION
**Pull request by Todmy**
I have checked all links and found out that some of them is broken/incorrect/invalid. It will be nice if somebody check removed by me links. Because it may be related for firewall issues or country restrictions.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -788,30 +760,20 @@
...
   - [Portfolio Site](http://corentinbac.com/)
-  - [Laravist](https://laravist.com/)
-  - [Atiiv](https://atiiv.com) - An app aimed for personal trainers and their clients.
   - [Statamic](https://statamic.com)
   - [Embalses!](http://embalses.azurewebsites.net/) - A tool to report water dam level using the U.S. Geological Survey database.
   - [TravelMap](http://clem.travelmap.fr) - A simple way for travelers to create a blog based on a Map.
   - [Proper Cloth Shirt Builder](https://propercloth.com/design-a-shirt) - Custom shirt builder.
-  - [CheckIt](https://check-it.io)
   - [Reddit News](https://github.com/Mati365/reddit-news) - A browser extension which show notifications and news from reddit.
-  - [Powerpuff Yourself by Cartoon Networks](https://www.powerpuffyourself.com/)
```
Comment (Scrum) [inline README.md]: @Todmy this page exists

File: README.md [hunk 2]
```diff
@@ -788,30 +760,20 @@
...
-  - [小桃酱](https://app.xiaotaojiang.com/)
-  - [cloudradioo](http://cloudradioo.com) - Web app that plays randomly all top 50 songs from the soundcloud charts
```
Comment (Scrum) [inline README.md]: this page exists, but not on english

File: README.md [hunk 3]
```diff
@@ -849,8 +810,6 @@
   - [1XBET](https://1xbet.com) - A betting company operating since 2007
   - [MyOwnTV](http://myowntv.org) - A streaming website for creating internet television
   - [CrowdCircus](https://crowdcircus.com) - Europe’s biggest crowdfunding- and crowdinvesting-aggregator
-  - [Talk to Someone](https://talktosomeone.net) - Free, anonymous and confidential online text chat with strangers around the world.
-  - [轮子工厂](http://www.wheelsfactory.cn) - A ui components and libraries sharing website
```
Comment (Scrum) [inline README.md]: this page exists, but not on english

File: README.md [hunk 4]
```diff
@@ -888,11 +846,9 @@
   - [27.ua](https://27.ua) - Ukraine-based internet hypermarket
   - [Chess Guardian](http://vitomd.com/vue-chess-guardian/) - Answer chess positional questions from your own games.
   - [Blackjack Break](https://blackjackbreak.com) - A quick game of blackjack
-  - [MECHANICAL](https://addons.mozilla.org/en-US/firefox/addon/mechmod/) - Reddit mod for Firefox that surfaces contextual data insights.
   - [GameVix](https://www.gamevix.com/) - Swap your used video game discs with others, hassle free. PWA with Material Design.
   - [VivifyScrum](https://app.vivifyscrum.com) - Agile project management app for teams that deliver. Customizable Scrum and Kanban boards.
   - [9GAG](https://9gag.com) - Popular online platform and social media website
-  - [CryptoVue](https://www.cryptovue.com/) - Real-time cryptocurrency dashboard
```
Comment (Scrum) [inline README.md]: page is but contains incomprehensible content

File: README.md [hunk 5]
```diff
@@ -919,9 +875,7 @@
   - [NAGA VIRTUAL](https://nagavirtual.com) - NAGA VIRTUAL is the first independent virtual goods market.
   - [Scroll.in](https://scroll.in) - Scroll.in is an independent news, information, and entertainment venture.
   - [Akunyi](https://akunyi.com) - Charity Website
-  - [What Would Mark Ruffalo Do?](https://www.whatwouldmarkruffalodo.com/) - Motivational website based on the story that Mark Ruffalo failed at several hundred auditions before finding success.
```
Comment (Scrum) [inline README.md]: @Todmy this page exists

File: README.md [hunk 6]
```diff
@@ -973,14 +927,11 @@
 
 ### Interactive Experiences
 
- - [Jean-Pierre Morin | 1700 LAPOSTE](http://1700laposte.com/jean-pierre-morin/)
- - [Facebook NewsFeed](https://newsfeed.fb.com/)
```
Comment (Scrum) [inline README.md]: redirect to main page facebook, not sure what it really needs

File: README.md [hunk 7]
```diff
@@ -253,7 +250,6 @@
 - [JavaScript Air 016 (03-30-2016)](https://javascriptair.com/episodes/2016-03-30/)
 - [Codecasts #2 - Falando Sobre Vuejs e Web Components (2016-08-19) [pt-BR]](https://soundcloud.com/codecasts/2-falando-sobre-vuejs-e-web-components)
 - [Full Stack Radio #50 (09-21-2016)](http://www.fullstackradio.com/50)
-- [和 Vue.js 框架的作者聊聊前端框架开发背后的故事 [zh-CN]](http://teahour.fm/2015/08/16/vuejs-creator-evan-you.html)
```
Comment (Scrum) [inline README.md]: @Todmy fix in #3095

File: README.md [hunk 8]
```diff
@@ -788,30 +760,20 @@
...
   - [Statamic](https://statamic.com)
   - [Embalses!](http://embalses.azurewebsites.net/) - A tool to report water dam level using the U.S. Geological Survey database.
   - [TravelMap](http://clem.travelmap.fr) - A simple way for travelers to create a blog based on a Map.
   - [Proper Cloth Shirt Builder](https://propercloth.com/design-a-shirt) - Custom shirt builder.
-  - [CheckIt](https://check-it.io)
   - [Reddit News](https://github.com/Mati365/reddit-news) - A browser extension which show notifications and news from reddit.
-  - [Powerpuff Yourself by Cartoon Networks](https://www.powerpuffyourself.com/)
-  - [小桃酱](https://app.xiaotaojiang.com/)
-  - [cloudradioo](http://cloudradioo.com) - Web app that plays randomly all top 50 songs from the soundcloud charts
-  - [vNotes](https://halfeld.github.io/v-notes/#!/) - Simple and beautiful notepad to Markdown with Vue.js and Local Storage API.
```
Comment (Scrum) [inline README.md]: @Todmy need change to https://github.com/IgorHalfeld/v-notes

File: README.md (hunk 4 above)
Comment (Todmy) [reply README.md]: yes. So Do we need it?

File: README.md (hunk 6 above)
Comment (Todmy) [reply README.md]: so? Do we need it?

File: README.md (hunk 4 above)
Comment (Scrum) [reply README.md]: @Todmy I don’t think it’s worth leaving, because not enough information

File: README.md (hunk 6 above)
Comment (Scrum) [reply README.md]: @Todmy No

**Pull request merged**